src/applications/vaos/utils/constants.js

```diff
@@ -493,14 +493,21 @@ export const TRAVEL_CLAIM_MESSAGES = {
   success: 'Data retrieved successfully.',
 };
 
-export const DATE_FORMAT_STRINGS = {
+export const DATE_FORMATS = {
```

Comment:
I found `DATE_FORMAT_STRINGS` a little verbose so I shortened it. Let me know if the longer form is prefered.